workspaces/azureml_devtestlabs/readme.md

```diff
@@ -0,0 +1,46 @@
+# Azure ML Worksapce
+
+This deploys a TRE workspace with a private Azure ML deployment.
+
+## Firewall Rules
+
+Please be aware that the following Firewall rules are opened for the workspace:
+
+URLs:
+
+- graph.windows.net
+- ml.azure.com
+- login.microsoftonline.com
+- aadcdn.msftauth.net
+- graph.microsoft.com
+- management.azure.com
+- *privatelink.api.azureml.ms
+- *privatelink.file.core.windows.net
+- *privatelink.vaultcore.azure.net
+- *privatelink.notebooks.azure.net
+- *privatelink.azurecr.io
+- *privatelink.blob.core.windows.net
+- viennaglobal.azurecr.io
+
+Service Tags:
+
+- Storage.WestEurope
```

Comment:
What are the service tags and why do we reference west europe?

Reply:
Good catch.

@deniscep - does this need to match the deployed region?

Reply:
Made region on the tag dynamic.